app/src/main/java/com/sopt/carrot/presentation/home/Card.kt

```diff
@@ -0,0 +1,10 @@
+package com.sopt.carrot.presentation.home
+
+import androidx.annotation.DrawableRes
+
+data class Card(
+    val id:Int,
+    @DrawableRes val image: Int,
+    val title: String,
+    val salary: String
```

Comment:
server 에서 detail 에 대한 정보는 api 에서 제공되어지고 있지않아서 별도로 data class 에 추가하지 않은것같은데요~
nullable 하게 구성해보는건 어떨까요?

- null 처리 연습에 좋음
  - 서버에서는 정해진 스펙에 따라서 경우에 따라 null 이 넘어오는 경우도 있다.

```suggestion
    val salary: String,
    val detail: String? = null
```

Reply:
왕 대박 꿀팁 감사합니다ㅎㅎ 적용해볼게요~!